Elevel/Elevel.Api/Controllers/TopicController.cs

```diff
@@ -17,41 +17,36 @@ public TopicController()
         }
 
         [HttpPost]
-        public async Task<IActionResult> Create([FromBody]CreateTopicCommand.Request request)
+        public async Task<IActionResult> Create([FromBody] CreateTopicCommand.Request request)
         {
             return Ok(await Mediator.Send(request));
```

Comment:
What it should return if the Topic will not be created?
200 Ok?

Reply:
@andreyrogov8 In this case will throw exception. in this case need global exception handler. It is not in the scope of this task.